src/Common/tests/System/RealFormatterTestsBase.netcoreapp.cs

```diff
@@ -14,29 +14,118 @@ public abstract class RealFormatterTestsBase
         protected abstract string InvariantToStringDouble(double d, string format);
         protected abstract string InvariantToStringSingle(float f, string format);
 
+        [Theory]
+        [InlineData("#,###.00", double.Epsilon, ".00")]
+        [InlineData("#,###.00", double.MaxValue, "179,769,313,486,232,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000.00")]
+        [InlineData("#,###.00", Math.E, "2.72")]
+        [InlineData("#,###.00", Math.PI, "3.14")]
+        [InlineData("#,###.00", 0.0, ".00")]
+        [InlineData("#,###.00", 0.0046, ".00")]
+        [InlineData("#,###.00", 0.005, ".01")]
+        [InlineData("#,###.00", 0.125, ".12")]
+        [InlineData("#,###.00", 0.5, ".50")]
+        [InlineData("#,###.00", 0.51, ".51")]
+        [InlineData("#,###.00", 0.56789, ".57")]
+        [InlineData("#,###.00", 0.84551240822557006, ".85")]
+        [InlineData("#,###.00", 1.0, "1.00")]
+        [InlineData("#,###.00", 505.0, "505.00")]
+        [InlineData("#,###.00", 506.0, "506.00")]
+        [InlineData("#,###.00", 545.0, "545.00")]
+        [InlineData("#,###.00", 545.1, "545.10")]
+        [InlineData("#,###.00", 555.0, "555.00")]
+        [InlineData("#,###.00", 65747.125, "65,747.12")]
```

Comment:
An exception to the backwards compatible behavior is that the rounding behavior is correct. So, on `.NET Framework` this would have returned `65,747.13` instead.